sources/packages/backend/workflow/test/2025-2026/fulltime-assessment/costs/fulltime-assessment-costs-scholarship-bursary.e2e-spec.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  ZeebeMockedClient,
+  createFakeConsolidatedFulltimeData,
+  executeFullTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+
+describe(`E2E Test Workflow part-time-assessment-${PROGRAM_YEAR}-costs-scholarship-bursary.`, () => {
+  it(
+    "Should calculate $0 for total scholarship and bursaries when " +
+      "student declares that they did not receive any scholarship and bursaries.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataScholarshipAmount = null;
```

Comment:
`studentDataScholarshipAmount` is null and does not need to be set as `null again.`